### ISSUE TITLE
Assorted Mappings

### ISSUE THREAD

**Issue by purposelycryptic** [ScudLee/anime-lists#275]
```xml
  <anime anidbid="12689" tvdbid="272316" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="">
    <name>Non Non Biyori Repeat: Hotaru ga Tanoshinda</name>
    <mapping-list>
      <mapping anidbseason="0" tvdbseason="0">;1-2;</mapping>
    </mapping-list>
  </anime>
```

```xml
  <anime anidbid="13187" tvdbid="272316" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="">
    <name>Non Non Biyori: Vacation</name>
    <mapping-list>
      <mapping anidbseason="0" tvdbseason="0">;1-3;</mapping>
    </mapping-list>
  </anime>
```

```xml
  <anime anidbid="10240" tvdbid="movie" defaulttvdbseason="1" episodeoffset="" tmdbid="347182" imdbid="tt7230298">
    <name>Gekijouban Meiji Tokyo Renka</name>
  </anime>
```

```xml
  <anime anidbid="12461" tvdbid="355576" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
    <name>Meiji Tokyo Renka</name>
  </anime>
```

**Comment by purposelycryptic** [comment]
```xml
  <anime anidbid="12687" tvdbid="movie" defaulttvdbseason="" episodeoffset="" tmdbid="519811" imdbid="tt6069846">
    <name>Complex x Complex</name>
  </anime>
```

**Comment by purposelycryptic** [comment]
```xml
  <anime anidbid="12729" tvdbid="music video" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
    <name>3-D Tengoku</name>
  </anime>
```

**Comment by purposelycryptic** [comment]
Included in Pull Request #276 - "Mappings for Winter/Spring 2019 Anime"
